src/services/LessonPlanService.js

```diff
@@ -12,13 +13,24 @@ const LessonPlanService = {
    */
   deleteLessonPlan: async function (name) {
     try {
+      let favouritedPath = `${MAINDIRECTORY}/Favourited/${name}/`;
+      let defaultPath = `${MAINDIRECTORY}/Default/${name}/`;
+      let path;
+
+      // check if file exists, assigning appropriate path if so
+      if (await checkFileExists(favouritedPath)) {
+        path = favouritedPath;
+      } else if (await checkFileExists(defaultPath)) {
```

Comment:
Right, it is impossible that the lesson plan is in both favouritedPath and defaultPath